src/googleclouddebugger/python_breakpoint.py

```diff
@@ -253,15 +253,32 @@ def GetBreakpointId(self):
 
   def GetExpirationTime(self):
```

Comment:
Might be worth adding in that if there is no creation time, the expiration time will be in the past.